### PR TITLE
Defender and Crusher tail smash has a 1% chance of doing a BONK! sound instead.

### PR DESCRIPTION
**Pull request by Diegoflores31**
# About the pull request

Defender and Crusher tail smash has a 1% chance of doing a BONK! sound instead.

# Explain why it's good for the game
its one of those fun occasional things it will be cool for gameplay 

# Testing Photographs and Procedure
<details>
<summary>Screenshots & Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

# Changelog
:cl:
add: Defender Tail Slam has a 1% chance to do a BONK sound instead.
/:cl:

### CODE REVIEW

File: code/modules/mob/living/carbon/xenomorph/abilities/general_powers.dm [hunk 1]
```diff
@@ -985,7 +985,11 @@
 
 	if(blunt_stab)
 		stabbing_xeno.visible_message(SPAN_XENOWARNING("\The [stabbing_xeno] swipes its tail into [target]'s [limb ? limb.display_name : "chest"], bashing it!"), SPAN_XENOWARNING("You swipe your tail into [target]'s [limb? limb.display_name : "chest"], bashing it!"))
-		playsound(target, "punch", 50, TRUE)
+		var/playing_sound = rand(1, 99)
+		if(playing_sound == 99)
+			playsound(target, "bonk", 50, TRUE)
```
Comment (BeagleGaming1) [inline code/modules/mob/living/carbon/xenomorph/abilities/general_powers.dm:990]: "punch" is not a real sound, it just leads to a get_sfx switch where
`			if("punch")
S = pick('sound/weapons/punch1.ogg','sound/weapons/punch2.ogg','sound/weapons/punch3.ogg','sound/weapons/punch4.ogg')`

Comment (Diegoflores31) [reply code/modules/mob/living/carbon/xenomorph/abilities/general_powers.dm]: i forgor , i just fixed it

**Review by Zonespace27 — DISMISSED** [review]
where is this sound sourced from?

**Comment by Diegoflores31** [comment]
> where is this sound sourced from?

Youtube , pretty sure its a generic bonk

**Comment by Diegoflores31** [comment]
Bruh i pressed close with comment 💀

File: code/modules/mob/living/carbon/xenomorph/abilities/general_powers.dm [hunk 2]
```diff
@@ -985,7 +985,11 @@
 
 	if(blunt_stab)
 		stabbing_xeno.visible_message(SPAN_XENOWARNING("\The [stabbing_xeno] swipes its tail into [target]'s [limb ? limb.display_name : "chest"], bashing it!"), SPAN_XENOWARNING("You swipe your tail into [target]'s [limb? limb.display_name : "chest"], bashing it!"))
-		playsound(target, "punch", 50, TRUE)
+		var/playing_sound = rand(1, 99)
+		if(playing_sound == 99)
+			playsound(target, 'sound/effects/bonk.ogg', 50, TRUE)
```
Comment (realforest2001) [inline code/modules/mob/living/carbon/xenomorph/abilities/general_powers.dm:990]:
```suggestion
		if(prob(1))
			playsound(target, 'sound/effects/bonk.ogg', 50, TRUE)
```

**Review by realforest2001 — CHANGES_REQUESTED** [review]

**Review by Drulikar — APPROVED** [review]

**Pull request merged**